week-4/tic-tac-toe/utilities.py

```diff
@@ -0,0 +1,35 @@
+class utilities:
```

Comment:
we are just using classes for using functions and not for classes, we are never using self, we could have just used functions , why/ why not ? do you think we should have maintained classes to maintain grid ?